spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/config/DataFlowControllerAutoConfiguration.java

```diff
@@ -124,6 +125,7 @@ public MavenResourceLoader MavenResourceLoader(MavenProperties properties) {
 	}
 
 	@Bean
+	@ConditionalOnMissingBean
```

Comment:
Yes, it should be: `@ConditionalOnMissingBean(DelegatingResourceLoader.class)`, I must have used the same name since it worked when I tested the k8s deployer